numba_dpex/experimental/launcher.py

```diff
@@ -98,6 +109,27 @@ def insert_kernel_bitcode_as_byte_str(
             bytes=kernel_module.kernel_bitcode,
         )
 
+    def allocate_meminfos_array(self, num_meminfos):
+        """Allocates an array to store the LLVM Value for every kernel argument.
```

Comment:
Is it the kernel arg or the meminfo pointers for the kernel args that have a meminfo pointer?

Reply:
Meminfo pointers, as I said did not finish docstrings yet.